### PR TITLE
Change all IP address that are using Oranges IP range to RFC1918 address space

### LINKED ISSUE #17701: Use private address space for VIPs in tests
This test is using an IP address from Orange's public range to validate some functionality:

https://github.com/cilium/cilium/blob/da26d0ae6f25c1fa28ba1d686e5505eb245e4908/test/l4lb/test.sh#L97

There are other examples as well, eg in `test/runtime/lb.go`.

We should avoid using IP addresses that are allocated to other purposes in the testing code.

More appropriate address ranges would be, eg:
* RFC1918 address space (`10.0.0.0/8`, `192.168.0.0/16`, `172.16.0.0/12`)
* Maybe TEST-NET-1 (`192.0.2.0/24`)

### PR DESCRIPTION
**Pull request by duttaANI**
Please ensure your pull request adheres to the following guidelines:

- [x] For first time contributors, read [Submitting a pull request](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#submitting-a-pull-request)
- [x] All code is covered by unit and/or runtime tests where feasible.
- [ ] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue.
- [ ] All commits are signed off. See the section [Developer’s Certificate of Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#dev-coo)
- [ ] Provide a title or release-note blurb suitable for the release notes.
- [ ] Thanks for contributing!

Fixes: #17701

### CODE REVIEW

**Comment by pchaigno** [comment]
@duttaANI I'm happy to trigger the end-to-end tests but could you first rebase on master and sign off your commit?

File: daemon/cmd/policy_test.go [hunk 1]
```diff
@@ -699,7 +699,7 @@ func (ds *DaemonSuite) TestReplacePolicy(c *C) {
 			EgressCommonRule: api.EgressCommonRule{
 				ToCIDR: []api.CIDR{
 					"1.1.1.1/32",
```
Comment (brb) [inline daemon/cmd/policy_test.go]: Should we also then change the CF addrs? Those addrs are used as dummy, i.e. no actual traffic is sent, so I don't see a big problem with both.

**Review by pchaigno — CHANGES_REQUESTED** [review]
Please fix the commit history and we'll run the end-to-end tests. This branch's commit history currently has several merge commits which shouldn't be here and a commit authored by someone else who seems to have been picked up by mistake. The changes in this pull request should be split up into commits with changes that belong together (in some cases, that may be a single commit).

**Comment by duttaANI** [comment]
> Please fix the commit history and we'll run the end-to-end tests. This branch's commit history currently has several merge commits which shouldn't be here and a commit authored by someone else who seems to have been picked up by mistake. The changes in this pull request should be split up into commits with changes that belong together (in some cases, that may be a single commit).

I have created a new draft PR [here](https://github.com/cilium/cilium/pull/18029) which contains only single commit by me .

**Comment by pchaigno** [comment]
/test-1.22-4.19

**Review by pchaigno — APPROVED** [review]

**Review by rolinh — APPROVED** [review]

**Comment by qmonnet** [comment]
/test-1.22-4.19

**Review by qmonnet — APPROVED** [review]

**Review by jrajahalme — APPROVED** [review]

**Comment by pchaigno** [comment]
I think we can merge this without waiting for all reviews. The failing k8s-1.22-kernel-4.19 test is known flake https://github.com/cilium/cilium/issues/18014. We have already received four reviews and the changes are fairly trivial. Marking ready to merge.

**Review by nebril — APPROVED** [review]

**Review by ldelossa — APPROVED** [review]

**Pull request merged**